tests/libxc/dft-anion/input.dat

```diff
@@ -0,0 +1,121 @@
+   'M06-2X':  -76.1924951379999982,        #TEST
+     'VV10':  -76.3278854999999936,        #TEST
+    'PBEOP':  -76.1512636180000015,        #TEST
+ 'MGGA_MS2':  -76.2383272721999958,        #TEST
+   'MN12-L':  -76.1407199697000010,        #TEST
+   'N12-SX':  -76.1661886348999957,        #TEST
+     'dlDF':  -76.6884354622000046,        #TEST
+  'LRC-BOP':  -76.0672253225999953,        #TEST
+     'BP86':  -76.2381037258000021,        #TEST
+  'HCTH120':  -76.2200708894999934,        #TEST
+     'EDF1':  -76.2429501061999986,        #TEST
+  'LC-VV10':  -76.1371843019000067,        #TEST
+  'wB97M-V':  -76.1905787638000049,        #TEST
+   'M08-SO':  -76.1758524238999968,        #TEST
+ 'MGGA_MVS':  -76.1849932563000038,        #TEST
+  'revPBE0':  -76.1902459601000004,        #TEST
+    'M06-L':  -76.2104553281999983,        #TEST
+  't-HCTHh':  -76.2117681964000013,        #TEST
+  'HCTH407':  -76.2208420692000033,        #TEST
+'CAM-B3LYP':  -76.2006317957000050,        #TEST
+ 'MPW1PW91':  -76.2096624991999931,        #TEST
+  'MPW1B95':  -76.1891029766999992,        #TEST
+  'HCTH147':  -76.2248695524999960,        #TEST
+   'M06-HF':  -76.1952167047000017,        #TEST
+   'B3LYP5':  -76.1925957771999975,        #TEST
+     'PW91':  -76.2047640842999954,        #TEST
+   'wM05-D':  -76.1936388223000023,        #TEST
+    'B3LYP':  -76.2328643543000055,        #TEST
+      'PBE':  -76.1456556148000061,        #TEST
+   'MPWB1K':  -76.1823596442000053,        #TEST
+    'B97-K':  -76.1837967257999935,        #TEST
+      'B97':  -76.1968376328999994,        #TEST
+   'B1PW91':  -76.2117015982999959,        #TEST
+     'VSXC':  -76.2379777115999957,        #TEST
+   'BHHLYP':  -76.1821602865999949,        #TEST
+  'BP86VWN':  -76.2429496642999993,        #TEST
+     'EDF2':  -76.1918127816000066,        #TEST
+    'B97-2':  -76.1949367433000049,        #TEST
+  'wM06-D3':  -76.1968859696000038,        #TEST
+  'MPW1PBE':  -76.1849875645000054,        #TEST
+    'B97-1':  -76.1972144151999942,        #TEST
+  'wB97X-V':  -76.1899409795999958,        #TEST
+    'M11-L':  -76.2106761812000002,        #TEST
+ 'LRC-wPBE':  -76.1675024440999948,        #TEST
+    'PWB6K':  -76.2470965274000037,        #TEST
+   'PW6B95':  -76.3047778709999989,        #TEST
+   'MN15-L':  -76.1306121525000066,        #TEST
+ 'revTPSSh':  -76.1937528435999951,        #TEST
+'SOGGA11-X':  -76.1982273807000041,        #TEST
+  'MN12-SX':  -76.1521811698000022,        #TEST
+  'wB97X-D':  -76.1994640198000042,        #TEST
+  'MPW1LYP':  -76.1938387614000021,        #TEST
+    'PBE50':  -76.1459052103000005,        #TEST
+ 'MGGA_MS0':  -76.2427928287000043,        #TEST
+      'N12':  -76.1636902505999984         #TEST
+}                                          #TEST
+                                           
+for func in data.keys():                                   #TEST
+    try: func_energy = energy(func)                        #TEST
```

Comment:
The original try/except here was functionals that we did not implement. For actual tests I would very much prefer just commenting out the functionals that we do not have as it serve as a list of functional would should look at implementing and ensure that we cannot regress in functionality without noticing.

Reply:
Please see `tests/libxc` for a reasonable way of checking these cutoffs. The test must actually fail if a certain threshold is crossed.